packages/react-components/src/Popup.tsx

```diff
@@ -1,34 +1,122 @@
 // Copyright 2017-2021 @polkadot/react-components authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { PopupProps } from 'semantic-ui-react';
+import React, { useEffect } from 'react';
+import styled, { css } from 'styled-components';
 
-import React from 'react';
-import { Popup as SUIPopup } from 'semantic-ui-react';
+import { useToggle } from '@polkadot/react-hooks';
 
 interface Props {
-  children: React.ReactNode;
   className?: string;
-  isOpen?: boolean;
-  on?: PopupProps['on'];
-  onClose?: () => void;
-  position?: PopupProps['position'];
-  trigger?: React.ReactNode;
+  value?: React.ReactNode;
+  children?: React.ReactNode;
+  position?: 'left' | 'middle' | 'right'
+  onCloseAction?: () => void;
 }
 
-function Popup ({ children, className = '', isOpen, on = 'click', onClose, position = 'bottom right', trigger }: Props): React.ReactElement<Props> {
+const Popup: React.FC<Props> = ({ children, className = '', onCloseAction, position = 'left', value }): React.ReactElement | null => {
+  const [isOpen, toggleIsOpen] = useToggle(false);
+
+  useEffect(() => {
+    if (!isOpen && onCloseAction) {
+      onCloseAction();
+    }
+  }, [isOpen, onCloseAction]);
+
   return (
-    <SUIPopup
-      className={className}
-      on={on}
-      onClose={onClose}
-      open={isOpen}
-      position={position}
-      trigger={trigger}
-    >
-      {children}
-    </SUIPopup>
+    <PopupWrapper>
+      {isOpen && (
+        <PopupWindow position={position}>
+          { value }
+        </PopupWindow>
+      )}
+      <PopupTrigger
+        className={`ui--Popup ${className}`}
+        onClick={toggleIsOpen}
+      >
+        {children}
+      </PopupTrigger>
+    </PopupWrapper>
   );
-}
+};
 
 export default React.memo(Popup);
+
+const PopupWrapper = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+`;
+
+const PopupTrigger = styled.div`
+`;
+
+interface WindowProps {
+  position: 'left' | 'middle' | 'right'
+}
+
+const PopupWindow = styled.div<WindowProps>`
+  position: absolute;
+  margin-top: 0.7rem;
+  top: 100%;
+  //bottom: 0;
```

Comment:
not needed?

Reply:
I think it's not in the code anymore, you must have looked at some old commit